odkx-src/notify-install.rst

```diff
@@ -0,0 +1,79 @@
+Installing ODK-X Notify
+=======================
+
+.. _notify-prereqs:
+
+Prerequisites
+-------------------
+
```

Comment:
Are these the prerequisites for the tablet? Helpful if you state that here.

Reply:
Done.